index.js

```diff
@@ -10,3 +10,31 @@ mongoose.connect('mongodb://localhost/recipeApp', { useNewUrlParser: true })
     console.error('Error connecting to mongo', err);
```

Comment:
you can also need all your CRUD function inside the mongo connect and when they are all done you can disconnect your DB.